avocado/core/loader.py

```diff
@@ -613,7 +613,7 @@ def _find_avocado_tests(self, path):
         # The name used, in case of 'import avocado as avocadolib'
         mod_import_name = None
         # The resulting test classes
-        result = {}
+        result = collections.OrderedDict()
```

Comment:
I don't think we should define the order of discovered methods and if we do want to do it, I'd go with alphabetical order, which is used by unittest.